docs/topics/native/native-objc-interop.md

```diff
@@ -409,6 +409,47 @@ this library would disable these compiler checks.
 See [Interoperability with C](native-c-interop.md) for an example case where the library uses some plain C features,
 such as unsafe pointers, structs, and so on.
 
+## Experimental features
```

Comment:
Do we really need to group experimental features under the dedicated section?
We already have a partially experimental feature here: https://github.com/JetBrains/kotlin-web-site/blob/b9dd48d8ca233b72b92e13984cf6171dd1bd1781/docs/topics/native/native-objc-interop.md#suspending-functions
Generally, it might make sense to simply add stability status mentions to the feature descriptions instead.

Reply:
An explicit list of opt-ins makes it easier to distinguish a set of features that require some action from the user (passing an additional compiler flag in this case). Otherwise, it will make reading of the document harder because reader will have to constantly make "context switch" between opt-in and opt-out features.

Reply:
I don't think that it works in that way. When users come to the docs, they usually don't want to know which are experimental or not.
Otherwise, we should move suspend functions text to the Experimental section. I'd recommend eliminating the Experimental features section and adding a typical Experimental note (same as for suspend).